vulnerablecode/settings.py

```diff
@@ -28,7 +28,7 @@
 DEBUG = False
 
 ALLOWED_HOSTS = [
-    ".herokuapp.com",
+    os.environ.get("VC_ALLOWED_HOSTS", ".herokuapp.com"),
```

Comment:
`VC_ALLOWED_HOSTS` here is not great since you are providing a single host through the environ and not a list of hosts.
I would suggest a simple `VC_HOST` for this env variable.
Also, we probably want to remove the ".herokuapp.com" default.

Reply:
I think it is better to keep it near the django semantics, so I switched the variable to be splitable.

I removed herokuapp to switch to '*' for the default, as I am not sure we really want to get it not work by default.

Reply:
Now that `VC_ALLOWED_HOSTS` can be used to specify multiple hosts separated via a colon,  it solves both of the issues you guys pointed out ?